kernel/src/osquery.cpp

```diff
@@ -254,13 +257,7 @@ static int osquery_ioctl(dev_t dev, u_long cmd, caddr_t data,
 
     *(int *)e = test_counter;
     char *s = (char *)((int *)e + 1);
-    s[0] = 'H';
-    s[1] = 'E';
-    s[2] = 'L';
-    s[3] = 'L';
-    s[4] = 'O';
-    s[5] = '!';
-    s[6] = '\0';
+    memset(s, 'H', length);
```

Comment:
Real hackers use a [nopseld](https://www.corelan.be/index.php/2011/12/31/exploit-writing-tutorial-part-11-heap-spraying-demystified/#0x0c0c0c0c). Just saying.